test/color_test.py

```
@@ -989,6 +1070,20 @@ def test_cmy__all_elements_within_limits(self):
             self.assertTrue(0 <= m <= 1)
             self.assertTrue(0 <= y <= 1)
 
+    def test_oklab__all_elements_within_limits(self):
+        for c in rgba_combos_Color_generator():
+            l, a, b = c.oklab
```

Ruff (0.16.0)

[error] Ambiguous variable name: `l`

(E741)